WebDriverManager/DriverConfigs/Impl/ChromeConfig.cs

```diff
@@ -63,10 +72,49 @@ public virtual string GetBinaryName()
 
         public virtual string GetLatestVersion()
         {
-            return GetLatestVersion(LatestReleaseVersionUrl);
+            var chromeReleases = _chromeForTestingClient.GetLastKnownGoodVersions().Result;
+            var chromeStable = chromeReleases.Channels.Stable;
+
+            _chromeVersionInfo = new ChromeVersionInfo
+            {
+                Downloads = chromeStable.Downloads
+            };
+
+            return chromeStable.Version;
+        }
+
+        public virtual string GetMatchingBrowserVersion()
+        {
+            var rawChromeBrowserVersion = GetRawBrowserVersion();
+            if (string.IsNullOrEmpty(rawChromeBrowserVersion))
+            {
+                throw new Exception("Not able to get chrome version or not installed");
+            }
+
+            var chromeVersion = VersionHelper.GetVersionWithoutRevision(rawChromeBrowserVersion);
+
+            // Handle downloading versions of the chrome webdriver less than what's supported by the Chrome for Testing known good versions API
+            // See https://googlechromelabs.github.io/chrome-for-testing for more info
+            var matchedVersion = new Version(chromeVersion);
+            if (matchedVersion < MinChromeForTestingDriverVersion)
+            {
+                var url = ExactReleaseVersionPatternUrl.Replace("<version>", chromeVersion);
+                _chromeVersion = GetVersionFromChromeStorage(url);
+            }
+            else
+            {
+                _chromeVersion = GetVersionFromChromeForTestingApi(chromeVersion).Version;
```

Comment:
This worked for me, but I had to pass in rawChromeBrowserVersion instead of chromeVersion to get a match from the list (Version 115.0.5790.102).  This is because GetKnownGoodVersionsWithDownloads() returns versions as full versions (with revision numbers).

Reply:
Nice catch! Updated to use the `rawChromeBrowserVersion` variable which persists the revision number needed for Chrome for testing API's :)